cobigen-templates/templates-devon4j/src/main/java/com/devonfw/cobigen/templates/devon4j/utils/JavaUtil.java

```diff
@@ -490,4 +490,135 @@ public String getFirstEnumValue(String className) {
             return null;
         }
     }
+
+    /**
+     * Get the annotated table name of a given Entity class
+     *
+     * @param className
+     *            full qualified class name
+     * @return the annotated table name if existed or null
+     */
+    public String getEntityTableName(String className) {
```

Comment:
Add a test case for this method to src/test/java/com/devonfw/cobigen/templates/devon4j/test/utils/JavaUtilTest.java